functions/HttpTriggerAPIUsersIdTaskId/__init__.py

```diff
@@ -81,38 +138,45 @@ def connect():
         db_username = os.environ["ENV_DATABASE_USERNAME"]
         db_password = os.environ["ENV_DATABASE_PASSWORD"]
         driver = '{ODBC Driver 17 for SQL Server}'
+        logging.info('Uses hardcoded ODBC Driver 17 for SQL Server string')

```

Comment:
Do you think that this would more lean towards being a debug rather than an info?

Reply:
I am leaning towards info, as we are aware we're using this driver and it is hardcoded, so we may not want the message pop up each time for when we set the log levels at debug and up

Reply:
Updated per Toddy's comments